1-org/projects.tf

```diff
@@ -75,3 +75,26 @@ module "org_secrets" {
     application_name = "org-secrets"
   }
 }
+
+/******************************************
+  Project for SCC Notifications
+*****************************************/
+
+module "scc_notifications" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 8.0"
+  random_project_id           = "true"
+  impersonate_service_account = var.terraform_service_account
+  default_service_account     = "depriviledge"
+  name                        = "prj-scc-notifications"
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  folder_id                   = var.parent_folder
```

Comment:
According to the whitepaper the parent folder should be  `google_folder.common.id`